e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -664,68 +676,84 @@ Remove-Item -r $selfSignedCerts
 # Store all secrets in a KeyVault - Values will be pulled down from here to configure environment variables
 ###################################################################################################################################
 
-Write-Host "`nWriting secrets to KeyVault $keyVaultName."
```

Comment:
Now these are in a dictionary, with one conditionally added. All of them are added in a loop, so if we choose to change the kv command, we only need to change one line.